monitor/monitor.go

```diff
@@ -83,6 +83,7 @@ func NewMonitor(db *sql.DB, cfg *Config, logger *zerolog.Logger, apiUrl string)
 }
 
 func (m *Monitor) RunAll(wg *sync.WaitGroup, saveRawResponses bool) {
+	m.GetCoingeckoPrices()
```

Comment:
Probably better to run on it's own ticker -> we want this data to be fetched hourly or maybe a couple times a day.

This can be moved out or the `RunAll` and a new ticker can be added in `main.go`.